onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash 
+
+if [ ! "$1" = "8.0" ]
```

Comment:
why does 8.0 and 7.2 need to be hardcoded?

Reply:
https://github.com/onnx/onnx-tensorrt only has 7.2.1 as a branch option (not just 7.2)  
![image](https://user-images.githubusercontent.com/25233514/131719874-5ab60b42-86db-431d-87af-12fa8cea29cc.png)

Reply:
i meant it can be refactored to be passed as a parameter. 
the functionality also seems quite limited for it exist in its own shell script?
maybe refactor in future revision.